modules/intro/pages/release-process.adoc

```diff
@@ -0,0 +1,99 @@
+
+In addition to Full releases and Patch releases, occasionally TigerGraph will provide a One-Off patch for a selected set of existing releases that may or may not impact all customers.
+These are used to address security vulnerabilities or critical bugs in a timely manner.
+
+In summary, the TigerGraph release types are:
+
+. Full releases
+. Patch releases
+. One-Off patches
+
+This article describes the process we follow for these three types of releases.
+
+
+== Naming Convention
+For full and patch releases, we use https://en.wikipedia.org/wiki/Software_versioning[sequence-based identifiers] in the format *major.minor.patch*.
+
+Here, *major* is a number and increment for the most significant releases, *minor* is a number and increment for less significant releases, and *patch* is a number for the patch release.
+
+For example, in the release version *3.2.1*, the major version is 3, the minor version is 2, and the patch version is 1.
+
+Our documentation (found at https://docs.tigergraph.com/https://docs.tigergraph.com/[docs.tigergraph.com]) is updated with a new branch when the major or minor version number changes.
+We do not provide a new documentation branch for patch releases. Instead, we may update the corresponding *major.minor* version’s documents.
+
+== Release Types and Action Plan
+
+=== Full Release
+
+A full release is a planned non-trivial product release, where either the major or the minor number in the release version name is changed.
+Either the minor number is increased, leaving the major number unchanged, or the major number is increased and the minor number is reset to 0.
+
+A full release's name is usually *major.minor.0*, abbreviated as *major.minor*.
+It may include any of the following:
+
+* New features
+* Feature enhancements or updates
+* Bug fixes
+
+In addition, the following documentation deliverables accompany a full release:
+
+* xref:release-notes:index.adoc[] section included in the TigerGraph online documentation.
+* xref:installation:upgrade.adoc[Upgrade] section included in the TigerGraph online documentation.
+* Updated TigerGraph online documentation for each new full release version.
+
+*Notification:* Full releases will be announced on our documentation website.
+TigerGraph’s Customer Success team will also proactively share the release notes with existing customers.
+
+*Upgrades:* Customers using a legacy release can refer to the upgrade section of the online documentation to plan a server upgrade or contact TigerGraph Technical Support Engineering by opening a ticket at http://tigergraph.freshdesk.com[tigergraph.freshdesk.com] or emailing support@tigergraph.com.
+
+=== Patch Release
+
+A patch release is a release version that only includes fixes for the most recent one or two full releases.
+It will only be created when a set of critical bug fixes has accumulated that will affect a majority of customers before the next planned full release.
```

Comment:
Let's cut out "that will affect a majority of customers".  Including it sets up the negative idea that TigerGraph won't fix _my_ problem because I happen to be in the minority. It's enough to say the bugs are "critical", because our definition of _critical_ already embraces the impact to users in realistic settings.